helm/minio-operator/templates/operator-deployment.yaml

```diff
@@ -40,16 +40,9 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.operator.image.repository }}:{{ .Values.operator.image.tag }}"
           imagePullPolicy: {{ .Values.operator.image.pullPolicy }}
-          {{- if or .Values.operator.clusterDomain .Values.operator.nsToWatch }}
+          {{- with .Values.operator.env }}
           env:
-            {{- if .Values.operator.clusterDomain }}
-            - name: CLUSTER_DOMAIN
-              value: {{ .Values.operator.clusterDomain }}
-            {{- end }}
-            {{- if .Values.operator.nsToWatch }}
-            - name: WATCHED_NAMESPACE
-              value: {{ .Values.operator.nsToWatch }}
-            {{- end }}
+            {{ toYaml . | nindent 10 }}
```

Comment:
This is a breaking change how would this affect existing deployments with `helm`? @nitisht

Reply:
I'll add a deprecated notice in values.yaml and allow the current values to work for a while

Reply:
Done.